tests/utils/arrays.py

```diff
@@ -6,4 +6,9 @@
 
 
 def arrays_contain_same_elements(arr1: list[Any], arr2: list[Any]) -> bool:
-    return set(arr1) == set(arr2)
+    for item in arr1:
+        if item in arr2:
+            arr2.remove(item)
+        else:
+            return False
+    return not arr2
```

Comment:
Why was this changed?

Reply:
The old version didn't check that there was the same amount of each object

ex. [1,1,2] and [1,2] would be considered equal